gui/packages/desktop/src/main/index.js

```diff
@@ -25,6 +25,11 @@ const ApplicationMain = {
   _shouldQuit: false,
 
   run() {
+    // Since electron's GPU blacklists are broken, GPU acceleration won't work on older distros
+    if (process.platform == 'linux') {
```

Comment:
Please use `===` due to JS quirks we always ensure types too just to stay on the safe side.